### PR TITLE
Add App shortcuts

### PR DESCRIPTION
**Pull request by Elchi3**
Spec: https://w3c.github.io/manifest/#shortcuts-member

Chromiums: https://www.chromestatus.com/features/5059438092222464
Gecko: https://bugzilla.mozilla.org/show_bug.cgi?id=1582341
Webkit: https://bugs.webkit.org/show_bug.cgi?id=201964

I wasn't sure about Samsung, I will ask if someone knows.

### CODE REVIEW

**Comment by Elchi3** [comment]
Setting experimental to `true` since https://developer.mozilla.org/en-US/docs/Web/Manifest is marked experimental altogether (and I'm no PWA expert to judge if that's reasonable).

**Comment by Elchi3** [comment]
@AdaRoseCannon @lauramorinigo do you know if Samsung ships https://web.dev/app-shortcuts/ ?

Seems like there was interest to do so per https://github.com/w3c/manifest/pull/768#issuecomment-532941900

File: html/manifest/shortcuts.json [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+  "html": {
+    "manifest": {
+      "shortcuts": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/Manifest/shortcuts",
+          "spec_url": "https://w3c.github.io/manifest/#shortcuts-member",
+          "support": {
+            "chrome": {
+              "version_added": "85",
+              "notes": "Supported on Windows only."
```
Comment (ddbeck) [inline html/manifest/shortcuts.json:11]: The link to the Chrome status entry is broken, so I don't know the full story here, but I suspect this needs to be flagged as partial for https://github.com/mdn/browser-compat-data/blob/master/docs/data-guidelines.md#operating-system-limitations-imply-partial_implementation

Comment (Elchi3) [reply html/manifest/shortcuts.json:12]: Good catch! Added :)

**Comment by AdaRoseCannon** [comment]
I checked on Samsung Internet, with the Shortcuts Demo App and couldn't get any shortcuts to appear. So I don't think it is yet supported in Samsung Internet

**Comment by Elchi3** [comment]
Marked Samsung as false, thanks Ada!

@ddbeck ready for re-review

**Review by ddbeck — APPROVED** [review]
Thank you! 🎉

**Pull request merged**